test/sql/udf_function/udf_functions_to_test.hpp

```diff
@@ -0,0 +1,54 @@
+/*HEADER file with all UDF Functions to test*/
+#include "duckdb/common/types/string_type.hpp"
+
+#pragma once
+using namespace duckdb;
+
+//UDF Functions to test
+bool BOOL(bool a) {return a;}
```

Comment:
These names are really generic, please rename them to e.g. `udf_bool` and place them in the DuckDB namespace. The current names might give name conflict problems on e.g. Windows.